packages/storybook/playwright.config.ts

```diff
@@ -29,7 +29,7 @@ export default defineConfig({
     trace: 'on-first-retry',
   },
   expect: {
-    timeout: process.env.CI ? 10000 : 5000,
+    timeout: process.env.CI ? 30000 : 10000,
```

Comment:
In my point of view 30s is really long. Do you have some evidence about this number?

Reply:
No thats just based on experience in another project